default/python/AI/freeorion_tools/lazy_initializer.py

```diff
@@ -21,7 +20,6 @@ def unlock(self):
     def __call__(self, function: Callable):
         @wraps(function)
         def wrapper(*args, **kwargs):
-            print("use zzzxxcc3", fo.currentTurn())
```

Comment:
Oops, my bad.  
This is my favorite prefix for debug prints, it's easy to find it in logs.